doc/gui/extension/example_library/example_library.py

```diff
@@ -10,11 +10,17 @@
 # specific language governing permissions and limitations under the License.
 
 from taipy.gui.extension import Element, ElementLibrary, ElementProperty, PropertyType
+import base64
 
 
 class ExampleLibrary(ElementLibrary):
     def __init__(self) -> None:
         # Initialize the set of visual elements for this extension library
+
+        logo_path = self.get_resource("resources/taipy_logo.png")
```

Comment:
The best I think would be to change the path not to confuse users about what we're looking for and where.
I also rename the image file itself for simplicity.
```suggestion
        logo_path = self.get_resource("icons/logo.png")
```